model/pyproject.toml

```diff
@@ -15,18 +15,27 @@ dependencies = [
     "datasets==2.8.0",
     "deepspeed==0.7.7",
```

Comment:
I am currently training with newer version of deepspeed everywhere since new HF transformers version complaine abut old deepspeed version .. IMO we can safely update this to latest version.